src/server/pkg/worker/master.go

```diff
@@ -261,14 +265,20 @@ nextInput:
 
 // jobManager feeds datums to jobs
 func (a *APIServer) runJob(ctx context.Context, jobInfo *pps.JobInfo, pool *grpcutil.Pool) error {
+	jobStart := time.Now()
+	defer func() { protolion.Infof("running job (%v) took (%v)\n", jobInfo, time.Since(jobStart)) }()
+	protolion.Infof("running job (%v)\n", jobInfo)
 	pfsClient := a.pachClient.PfsAPIClient
 	objectClient := a.pachClient.ObjectAPIClient
 	ppsClient := a.pachClient.PpsAPIClient
 
 	jobID := jobInfo.Job.ID
 	var jobStopped bool
 	var jobStoppedMutex sync.Mutex
+	retryCount := 0
 	backoff.RetryNotify(func() error {
+		defer func() { retryCount += 1 }()
+		protolion.Infof("running job (%v), retry (%v)\n", jobInfo, retryCount)
```

Comment:
More redundant logging here, this has exactly the same info as on line 270 with an extra copy of the retry count that's already present in `JobInfo`.